search_that_hash/printing.py

```diff
@@ -66,7 +66,7 @@ def one_print(result, chash):
             for i in types:
                 to_make.append(i["name"])
 
-            self.push(chash, result, to_make)
+            Prettifier.push(chash, result, to_make)
```

Comment:
How does this work? Isn't it `self` since it's an method of the class it's in?

Reply:
Fixed

Reply:
No you didn't :)

Reply:
This still isn't fixed

Reply:
My bad, this is fixed now :)